### PR TITLE
Update including quiz html, css, js

### PR DESCRIPTION
**Pull request by dasbeerboot**
Showing off my outcome uhaha

### CODE REVIEW

File: firstpage.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <link href="https://fonts.googleapis.com/css?family=Martel+Sans:900|PT+Sans" rel="stylesheet">
     <br>
     <br>
-      <title>Puzzle Something :: Name anyways</title>
+      <title>Happy Birthday Benny</title>
```
Comment (stevechun) [inline firstpage.html]: Oh so sweet

File: quiz.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<html>
+        <link rel="stylesheet" href="quiz.css">
+    <head>
+            <link href="https://fonts.googleapis.com/css?family=Martel+Sans:900|PT+Sans" rel="stylesheet">
+    
+
+
```
Comment (stevechun) [inline quiz.html]: I only want a single linespace here

Comment (dasbeerboot) [reply quiz.html]: I don't get it... please explain it with a little more detail bitte T_T

**Comment by dasbeerboot** [comment]
Just added hella more. I know it's too much I don't even wanna look at it. Help...

File: question2.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Martel+Sans:900|PT+Sans" rel="stylesheet">
+    <br>
+    <br>
+      <title>Happy Birthday Benny</title>
+  </head>
+  
+  <body>
+      <div class="quizbox">
+          <h2>Q2. How much is Ben's favourite Easter egg?</h2>
+          <p><img class="b-egg" src="b-egg.png">  This is Ben's favourite type of Easter Egg.
```
Comment (makeajourney) [inline question2.html]: I wanna recommend separating the paragraphs.
in this case, you can separate 3 of paragraphs.

File: question2.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            He went to the shop to buy one but found that the Easter Eggs had been packed up in boxes of three like this:
+            <br>
+            <br>
+            <img class="b-eggs" src="b-eggs.png">
+            <br>
+            <br>
+            "How much does this one egg cost?" Ben asked the staff.
+          The staff answered, "Easy. It's just ..."
+          </p>
+          <label for="$1">
```
Comment (makeajourney) [inline question2.html]: you have to more care about indentation.
Good code is easy to read.

File: quiz.html (hunk 2 above)
Comment (makeajourney) [reply quiz.html]: there are three blank lines. it's better to leave an only single blank line

Comment (dasbeerboot) [reply quiz.html]: ohh danke

File: question1.html [hunk 5]
```diff
@@ -9,8 +9,8 @@
   
   <body>
       <div class="quizbox">
-          <h2>Q1. Solve the following question, and convert the answer into MD5 hash and submit the result</h2>
-          <br><br><br><br><br>
+          <p><h2>Q1. Solve the following question, and convert the answer into MD5 hash and submit the result</h2></p>
```
Comment (makeajourney) [inline question1.html]: Well, Adding the p tag where out of h2 tag is unnecessary.

Comment (dasbeerboot) [reply question1.html]: o0o I wanted to have more space without adding 5 <br> so I decided to just use p tag... Is there any better way to have more space between sentences without using too many <br>

**Pull request merged**